platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/bls/message/AbstractMessage.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.bls.message;
+
+import static com.swirlds.common.utility.CommonUtils.throwArgNull;
+
+import com.swirlds.common.system.NodeId;
+
+/** A class representing a protocol message in the abstract */
+public abstract class AbstractMessage implements ProtocolMessage {
```

Comment:
My personal preference is that the abstract class be exactly equal to the interface name, but with `Abstract` prepended. Here you abbreviate it. When looking at only the file names of the interface and the abstract implementation, it's not immediately obvious that they are related (unless you have contextual knowledge).

Also, hould we consider adding `Bls` to class names like this? E.g.  In isolation, it's not immediately obvious that this is a BLS construct, but it would be handy if it was. In general, my strategy is to make it obvious what a class does given only its name.

Possible names: `BlsProtocolMessage` and `AbstractBlsProtocolMessage`.

If you take this suggestion, perhaps good to also reconsider file names for all the concrete implementations as well.

Reply:
I renamed these, as well as a number of other classes, to be consistent and more descriptive